projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.html

```diff
@@ -1,13 +1,35 @@
-<a
-  *ngFor="let item of pages"
-  [class]="item.type"
-  [class.disabled]="isInactive(item)"
-  [class.current]="isCurrent(item)"
-  [routerLink]="pageRoute"
-  [queryParams]="getQueryParams(item)"
-  [tabIndex]="isInactive(item) ? -1 : 0"
-  (click)="pageChange(item)"
-  [attr.aria-label]="getAriaLabel(item.label, item.type)"
+<!-- TODO: (CXSPA-7289) - Remove feature flag next major release -->
+<ng-container *cxFeature="'a11ySemanticPaginationLabel'"
```

Comment:
It probably would be best to have it. Since we are calling a new method with the feature flag enabled we would leave the old version without a proper aria-label.